package/MDAnalysis/analysis/base.py

```diff
@@ -281,25 +295,31 @@ def run(self, start=None, stop=None, step=None, verbose=None):
             stop frame of analysis
         step : int, optional
             number of frames to skip between each analysed frame
+        frames : array_like, optional
+            array of integers or booleans to slice trajectory
```

Comment:
We'll need some documentation / examples for this.

Reply:
Added an example here with two uses of 'frames'